types/htmlparser2/index.d.ts

```diff
@@ -3,7 +3,9 @@
 // Definitions by: James Roland Cabresos <https://github.com/staticfunction>
 //                 Linus Unnebäck <https://github.com/LinusU>
 //                 Johan Davidsson <https://github.com/johandavidson>
+//                 GP <https://github.com/paambaati>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
I'm not convinced we should be locking this to TypeScript 2.2, e.g I know at least one place it's being used with TypeScript 3.x.

Reply:
@WillGibson This is the minimum version of TS that these typings are guaranteed to work with. It works fine on TS 3.x.

Reply:
Ah, cool.